### PR TITLE
AlphaNet: enable stateproof recoverability

### PR DESCRIPTION
**Pull request by id-ms**

### CODE REVIEW

File: config/consensus.go [hunk 1]
```diff
@@ -1298,6 +1298,18 @@ func initConsensusProtocols() {
 	vAlpha5.ApprovedUpgrades = map[protocol.ConsensusVersion]uint64{}
 	Consensus[protocol.ConsensusVAlpha5] = vAlpha5
 	vAlpha4.ApprovedUpgrades[protocol.ConsensusVAlpha5] = 10000
+
+	vAlpha6 := vAlpha5
+	vAlpha6.ApprovedUpgrades = map[protocol.ConsensusVersion]uint64{}
+	Consensus[protocol.ConsensusVAlpha6] = vAlpha6
+	vAlpha5.ApprovedUpgrades[protocol.ConsensusVAlpha6] = 10000
```
Comment (onetechnical) [inline config/consensus.go:1305]: Shorten this down to 1000 or 5000?

File: config/consensus.go [hunk 2]
```diff
@@ -1298,6 +1298,18 @@ func initConsensusProtocols() {
...
+
+	vAlpha7 := vAlpha5
+	vAlpha7.StateProofUseTrackerVerification = true
+	vAlpha7.EnableCatchpointsWithSPContexts = true
+	vAlpha7.ApprovedUpgrades = map[protocol.ConsensusVersion]uint64{}
+	Consensus[protocol.ConsensusVAlpha7] = vAlpha7
+	vAlpha6.ApprovedUpgrades[protocol.ConsensusVAlpha7] = 10000
```
Comment (onetechnical) [inline config/consensus.go:1312]: This too, 1000 or 5000 sounds good.

File: config/consensus.go (hunk 1 above)
Comment (algorandskiy) [reply config/consensus.go:1305]: you have to modify other params to make is less than 10k

Comment (onetechnical) [reply config/consensus.go:1305]: Oh, so we need 10K voting, 10k upgrade vAlpha6, 10k voting, 10k upgrade vAlpha7, i.e. we need 40K rounds to complete before we can test?

**Review by algorandskiy — APPROVED** [review]

**Pull request merged**